nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/RelationCommentFeature.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.platform.comment;
+
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.RunnerFeature;
+
+/**
+ * Feature that provides the {@link org.nuxeo.ecm.platform.comment.impl.CommentManagerImpl}.
+ * 
+ * @since 11.1
+ * @deprecated since 10.3, in order to follow the service deprecation
```

Comment:
Sound weird, though I believe it's because the refactoring is added in 11.1 and used to test already deprecated code, right?

Reply:
yes exactly, what you explained, and to give enough information, with Kevin we decided to add the sentence `in order to follow the service deprecatio...` :)